tests/test_translate.py

```diff
@@ -365,7 +374,11 @@ def test_parallel_savepoint(
         with subtests.test(varname=varname):
             failing_names.append(varname)
             assert success(
-                output[varname], ref_data[varname][0], testobj.max_error, near0
+                output[varname],
+                ref_data[varname][0],
+                testobj.max_error,
+                near0,
+                testobj.near_zero,
```

Comment:
it might make sense to change the near0 variable name (and above where it is assigned) to ignore_near0 so it is more clear that this is the boolean argument, while testobj.near_zero is the value. it is fine to leave as-is, however.